net/hcxdumptool/Makefile

```diff
@@ -12,8 +12,8 @@ PKG_NAME:=hcxdumptool
 PKG_VERSION:=5.1.7
 PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://github.com/github.com/ZerBea/hcxdumptool/releases/download/v$(PKG_VERSION)
+PKG_SOURCE:=$(PKG_VERSION).tar.gz
```

Comment:
Absolutely not. Use codeload for this.

Reply:
Thanks! I wasn't aware of codeload.
Fixed in last commits.